### PR TITLE
Fix CSI2 Raw Conversion Artifacts, Improve CSI2 Example

### PR DESCRIPTION
**Pull request by Jake-Carter**
This PR fixes https://github.com/MaximIntegratedAI/MAX78000_SDK/issues/84

Also fixes https://github.com/MaximIntegratedAI/MAX78000_SDK/pull/83

The majority of this fix has been in the linkerfile, which now properly arranges the CSI2 raw buffers into sysram4.

The example now also features the improved console from the ImgCapture example with a limited command set (`help`, `reset`, `capture`)

Resolves MSDK-956

### CODE REVIEW

**Review by lorne-maxim — APPROVED** [review]

File: Examples/MAX78002/CSI2/utils.h [hunk 1]
```diff
@@ -30,8 +30,8 @@
  * ownership rights.
  *
  ******************************************************************************/
-#ifndef EXAMPLES_MAX78002_CSI2_UTILS_H_
-#define EXAMPLES_MAX78002_CSI2_UTILS_H_
+#ifndef _UTILS_H_
+#define _UTILS_H_
```
Comment (sihyung-maxim) [inline Examples/MAX78002/CSI2/utils.h:34]: Surprised the linter didn't complain.

**Review by sihyung-maxim — APPROVED** [review]
Looks good. Is the Image in utils supposed to be black?

File: Examples/MAX78002/CSI2/main.c [hunk 2]
```diff
@@ -184,29 +230,17 @@ int main(void)
     printf("python grab_image.py [COM#] [baudrate]\n");
```
Comment (sihyung-maxim) [inline Examples/MAX78002/CSI2/main.c:230]: Forgot to comment on this. The printf statements should be updated since grab_image.py was removed.

**Review by sihyung-maxim — CHANGES_REQUESTED** [review]

File: Examples/MAX78002/CSI2/utils.h (hunk 1 above)
Comment (Jake-Carter) [reply Examples/MAX78002/CSI2/utils.h]: A lot of files didn't change, so the linter skipped them.  Just fixed in all linter errors

**Review by sihyung-maxim — APPROVED** [review]

**Pull request merged**